packages/core-flows/src/definition/cart/steps/update-carts.ts

```diff
@@ -1,61 +1,44 @@
 import { ModuleRegistrationName } from "@medusajs/modules-sdk"
-import {
-  CartDTO,
-  FilterableCartProps,
-  ICartModuleService,
-  UpdateCartDataDTO,
-} from "@medusajs/types"
+import { CartWorkflow, ICartModuleService } from "@medusajs/types"
 import { getSelectsAndRelationsFromObjectArray } from "@medusajs/utils"
 import { StepResponse, createStep } from "@medusajs/workflows-sdk"
 
-type UpdateCartsStepInput = {
-  selector: FilterableCartProps
-  update: UpdateCartDataDTO
-}
-
 export const updateCartsStepId = "update-carts"
 export const updateCartsStep = createStep(
   updateCartsStepId,
-  async (data: UpdateCartsStepInput, { container }) => {
-    const service = container.resolve<ICartModuleService>(
+  async (data: CartWorkflow.UpdateCartWorkflowInputDTO, { container }) => {
```

Comment:
thought(non-blocking): Can we keep this as a bulk step? Unless there is a specific reason for converting it to a single cart update, I'd like that.